Blog: What makes Files different? (#767)
files-community/Website: src/routes/blog/posts/comparison/+page.md

---
title: What makes Files different?
description: An overview of the key differences of Files and File Explorer

> thumbnail: /blog-resources/comparison/FilesScreenshot.png

Reviewer: ```suggestion
thumbnail: /docs-resources/HeroImage.jpg
```
I know it's in the docs folder, but we might as well reuse it.

date: 02/21/2025
author: Lamparter
---

As the built-in File Explorer has seen some changes in recent version of Windows, it's interesting to consider what makes Files different from Windows File Explorer.

Recent updates to File Explorer in Windows 11 have brought significant improvements, including a refreshed design and the long-awaited addition of tabs. Meanwhile, Files has consistently provided a modern interface and tabbed navigation for users on both Windows 10 and Windows 11.

> However, what truly sets Files apart goes beyond tabs. While this feature has gained popularity, it was never meant to be the sole defining feature. Our aim was to create a modern file manager with an intuitive UX in collaboration with the community—an ambitious goal. Although there are still many exciting features we haven’t yet explored, Files stands out from File Explorer and other file managers through a range of unique differences, not just its tabs and modern styling.

Reviewer: ```suggestion
However, what truly sets Files apart goes beyond tabs. While this feature has gained popularity, it was never meant to be the sole defining feature. Our aim was to create a modern file manager with an intuitive UX in collaboration with the community. Although there are still many exciting features we haven’t yet explored, Files stands out from File Explorer and other file managers through a range of unique differences, not just its tabs and modern styling.
```

> So, here are a few key differences that make Files what it is:

Reviewer: Note to self: Currently up to reviewing this section of the PR.

> So, here are a few key differences that make Files what it is:
> 
> - Available on Windows 10: Files is backwards compatible down to Windows 10, whereas Explorer's newer functionality is only available on Windows 11.

Reviewer: ```suggestion
Files distinguishes itself in numerous ways, and as it continues to improve, we’re committed to maintaining backward compatibility with Windows 10. This ensures that all the great features and benefits of Files are accessible to more users, not just those running the latest version of Windows 11.
```

> - A consistent dark theme: Files uses the latest technologies and a powerful theming engine that makes it heavily customisable.

Reviewer: ```suggestion
Another area where Files stand out is in its design and user friendly experience. Built from the ground up with the latest UI frameworks from Microsoft, it looks and feels like a natural extension of Windows. This attention to detail extends beyond the main interface, Files maintains a consistent look and feel across the app, including a proper dark mode. Additionally, it supports custom themes and background images, allowing users to personalize their experience.
```

---
title: What makes Files different?
description: An overview of the key differences of Files and File Explorer
thumbnail: /blog-resources/comparison/Hero.jpg
date: 02/21/2025
author: Lamparter
---

> As the built-in File Explorer gets more features and newer File Explorer alternatives release, it's interesting to consider what makes Files different from the Windows File Explorer.

Reviewer: ```suggestion
As the built-in File Explorer has seen some changes in recent version of Windows, it's interesting to consider what makes Files different from Windows File Explorer.
```

In the recent years, File Explorer has had some major changes, including tabs, more consistent WinUI styles, and the experimental Pizza rewrite of File Explorer in the Windows App SDK.

When tabs were introduced to File Explorer in 2022, many members of the Files Community questioned the future of Files. As the lead developer of Files put it:

> The simple answer is that tabs was never the main point of Files, in fact, it wasn't even a feature for the first couple releases.

> And it's true - there are many differences that make Files an entirely different app than File Explorer, not just tabs and WinUI styles; what makes Files unique is the focus on usable design and features that make it easier to get work done.

Reviewer: ```suggestion
Recent updates to File Explorer in Windows 11 have brought significant improvements, including a refreshed design and the long-awaited addition of tabs. Meanwhile, Files has consistently provided a modern interface and tabbed navigation for users on both Windows 10 and Windows 11.

However, what truly sets Files apart goes beyond tabs. While this feature has gained popularity, it was never meant to be the sole defining element. Our aim was to create a sophisticated, modern file manager in collaboration with the community—an ambitious goal. Although there are still many exciting features we haven’t yet explored, Files stands out from File Explorer and other file managers through a range of unique differences, not just its tabs and WinUI styling.
